code/modules/paperwork/paper.dm

```diff
@@ -339,10 +339,11 @@
 
 	add_fingerprint(user)
 
-/obj/item/weapon/paper/fire_act(exposed_temperature, exposed_volume)
+/obj/item/weapon/paper/fire_act()
```

Comment:
Keep the arguments consistent with the parent proc even if you're not using them please.